### DISCUSSION TITLE
Electrolytic capacitors

### DISCUSSION THREAD

**Discussion by cgudrian** [horizon-eda/horizon#783] [Q&A] [answered]
Hello!

I cannot seem to find any electrolytic capacitors in the available pools. Aren't there really any or am I missing something?

Christian

**Comment by katyo** [comment, marked as answer]
@cgudrian As I see the corresponding entity exists but currently no parts which uses it
![image](https://github.com/user-attachments/assets/5b3844df-3481-43d7-adb1-a2248af4446d)

**Reply by cgudrian** [reply]
Thanks for the reply. I was wondering if I had overlooked something.
